infrastructure/main.tf

```diff
@@ -55,6 +55,14 @@ data "vault_generic_secret" "appeal_email_to" {
   path = "secret/${var.infrastructure_env}/sscs/appeal_email_to"
 }
 
+data "vault_generic_secret" "robotics_email_from" {
+  path = "secret/${var.infrastructure_env}/sscs/robotics_email_from"
+}
+
+data "vault_generic_secret" "robotics_email_to" {
+  path = "secret/${var.infrastructure_env}/sscs/robotics_email_to"
+}
+
```

Comment:
If we are going to use the same email from and to as Email pdf, then above two declaration is not needed.

Reply:
i disagree about the vault value -- 
even if they are the same now, independent keys make the featured decoupled and clear about what they are being used for